ERCS/erc-7540.md

```diff
@@ -56,7 +56,7 @@ Asynchronous deposit Vaults MUST override the ERC-4626 specification as follows:
 Asynchronous redeem Vaults MUST override the ERC-4626 specification as follows:
 
 1. The `redeem` and `withdraw` methods do not transfer `shares` to the Vault, because this already happened on `requestRedeem`. 
-2. The `owner` field of `redeem` and `withdraw` MUST be `msg.sender` unless the `owner` has approved the `msg.sender` as an operator.
+2. The `shareOwner` field of `redeem` and `withdraw` MUST be `msg.sender` unless the `shareOwner` has approved the `msg.sender` as an operator.
```

Comment:
Should be `requestOwner` here because you are claiming the request of this user
```suggestion
2. The `requestOwner` field of `redeem` and `withdraw` (defined as the `owner` in [ERC-4626](./eip-4626.md)) MUST be `msg.sender` unless the `requestOwner` has approved the `msg.sender` as an operator.
```
```suggestion
2. The `shareOwner` field of `redeem` and `withdraw` MUST be `msg.sender` unless the `shareOwner` has approved the `msg.sender` as an operator.
```